content/docs/command-reference/run.md

```diff
@@ -106,10 +106,10 @@ Relevant notes:
   also means that the stage command needs to recreate any directory structures
   defined as outputs every time its executed by DVC.
 
-- In some situations we have executed a stage and later notice that some of the
-  output files or directories it creates, which are already in the workspace,
-  are missing from `dvc.yaml` (`outs` field). We can
-  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+- In some situations, we have previously executed a stage, and later notice that
+  some of the files/directories are missing from the `deps` or `outs` fields of
```

Comment:
do we need to mention `outs` and `deps` fields here? 

also consider reverse order of the sentences.

Reply:
mentioning deps/outs fields was just a shorter alternative to "files/directories used by the stage as dependencies or created as outputs are missing from". BTW @imhardikj how we have different text in commit vs here in run. We have to pick one 😅

Reply:
BTW @imhardikj how we have different text in commit vs here in run. We have to pick one 😅

Reply:
Updated. Similar now.